docs/intro/ff-ui/toolbar.md

```diff
@@ -0,0 +1,234 @@
+
+Snapshots
+
+Snapshots are automatically saved as you build.
+
+:::info
+
+Users on the *Free* and *Standard* plans can access only one day of snapshots.
+
+:::
+
+### 6.1 Saving a version
+
+Saving the latest changes will store the current state of your app in the version control. In the future, if anything goes wrong, you can restore your app to the currently saved state.
+
+Ideally, you should save the version whenever you accomplish something important, for example, after finishing a page, adding configuration, etc.
+
+<div style={{
+    position: 'relative',
+    paddingBottom: 'calc(56.67989417989418% + 41px)', // Keeps the aspect ratio and additional padding
+    height: 0,
+    width: '100%'
+}}>
+    <iframe 
+        src="https://demo.arcade.software/d1rO0YLeMsrIt3Alb70E?embed&show_copy_link=true"
+        title="Sharing a Project with a User"
+        style={{
+            position: 'absolute',
+            top: 0,
+            left: 0,
+            width: '100%',
+            height: '100%',
+            colorScheme: 'light'
+        }}
+        frameborder="0"
+        loading="lazy"
+        webkitAllowFullScreen
+        mozAllowFullScreen
+        allowFullScreen
+        allow="clipboard-write">
+    </iframe>
+</div>
+### 6.2 Restoring a version
+
+Restoring the previous version will preserve the current version, then load the changes from the version you're restoring. Before restoration, you may want to view the changes in the previous version. To do this, select the Peek option, which opens the previous version in a new tab.
+
+![Screenshot 2024-05-06 at 10.57.51 AM.png](https://prod-files-secure.s3.us-west-2.amazonaws.com/09694bda-16f5-490f-aaac-3601b64ca6a2/62cabaee-ff75-496b-be7e-6df40a3cd8f2/Screenshot_2024-05-06_at_10.57.51_AM.png)
+
+## 7. Project comments
+
+This helps you leave your thoughts, opinions, questions, or feedback on a specific widget with your project team or a client. While adding a comment, you can tag users, and they will be able to respond, creating a thread of conversation.
+
+:::info
+
+To tag users, select the @ symbol and choose the project team member(s).
+
+:::
+
+## 8. Optimizations & Enhancements
+
+We scan and suggest enhancements to elevate your app's design and speed. Imagine having a personal consultant for design and performance dedicated to improving the user experience of your app.
+
+Optimizations: We spotlight elements that may slow down your app. For example, a query on a column, unused and duplicate backend query, a widget with unbounded size, etc.
+
+UI Enhancements: We offer tips for a more visually appealing and user-friendly design, like resizing a small tap target of any widget.
+
+:::info
+
+You can control what kind of suggestions you would like to receive by clicking on the settings icon on the right.
+
+:::
+
+![Optimization.avif](imgs/Optimization.avif)
+
+## 9. Project issues
```

Comment:
Can you please mention the difference between errors and warnings, conceptually and also in terms of compilation. Can be a short text.